airbyte-integrations/connectors/source-datadog/source_datadog/schemas/monitors.json

```diff
@@ -0,0 +1,365 @@
+              "type": ["array", "null"]
+            },
+            "metric": {
+              "type": ["string", "null"]
+            },
+            "type": {
+              "type": ["string", "null"]
+            }
+          },
+          "type": ["object", "null"]
+        },
+        "device_ids": {
+          "items": {
+            "type" : "string"
+          },
+          "type": ["array", "null"]
+        },
+        "enable_logs_sample": {
+          "type": ["boolean", "null"]
+        },
+        "enable_samples": {
+          "type": ["boolean", "null"]
+        },
+        "escalation_message": {
+          "type": ["string", "null"]
+        },
+        "evaluation_delay": {
+          "type": ["integer", "null"]
+        },
+        "group_retention_duration": {
+          "type": ["string", "null"]
+        },
+        "groupby_simple_monitor": {
+          "type": ["boolean", "null"]
+        },
+        "include_tags": {
+          "type": ["boolean", "null"]
+        },
+        "locked": {
+          "type": ["boolean", "null"]
+        },
+        "new_group_delay": {
+          "type": ["integer", "null"]
+        },
+        "new_host_delay": {
+          "type": ["integer", "null"]
+        },
+        "no_data_timeframe": {
+          "type": ["integer", "null"]
+        },
+        "notification_preset_name": {
+          "type": ["string", "null"]
+        },
+        "notify_audit": {
+          "type": ["boolean", "null"]
+        },
+        "notify_by": {
+          "items": {
+            "type": "string"
+          },
+          "type": ["array", "null"]
+        },
+        "notify_no_data": {
+          "type": ["boolean", "null"]
+        },
+        "on_missing_data": {
+          "type": ["string", "null"]
+        },
+        "renotify_interval": {
+          "type": ["integer", "null"]
+        },
+        "renotify_occurrences": {
+          "type": ["integer", "null"]
+        },
+        "renotify_statuses": {
+          "items": {
+            "type": "string"
+          },
+          "type": ["array", "null"]
+        },
+        "require_full_window": {
+          "type": ["boolean", "null"]
+        },
+        "scheduling_options": {
+          "additionalProperties": true,
+          "properties": {
+            "evaluation_window": {
+              "additionalProperties": true,
+              "properties": {
+                "day_starts": {
+                  "type": ["string", "null"]
+                },
+                "hour_starts": {
+                  "type": ["string", "null"]
+                },
+                "month_starts": {
+                  "type": ["integer", "null"]
+                }
+              },
+              "type": ["object", "null"]
+            }
+          },
+          "type": ["object", "null"]
+        },
+        "silenced": {
+          "patternProperties": {
+            ".+": {
+              "type" : ["integer", "null"]
+            }
+          }
```

Comment:
Not sure if this is supported.

Reply:
Yea I was a bit unsure what to do here. In the documentation it shows it can dynamically output properties here and this seemed to be the best way to represent it. In my testing it does seem to recognize it to an extent. It creates a `jsonb` column and seems to populate some data. Is there a better way to represent this?

![image](https://github.com/airbytehq/airbyte/assets/1613263/adeec345-7723-4aa0-98d8-4d60db7a774a)

https://docs.datadoghq.com/api/latest/monitors/